docs/developer/creating-test-environment.md

```diff
@@ -1,4 +1,4 @@
-# Set up the test environment
+# Set up test environment for local testing
```

Comment:
```suggestion
# Set up test environment for local dual-cluster mode
```

Reply:
I think this documentation still addresses local testing for single cluster cases